dist/twine.js

```diff
@@ -144,6 +146,14 @@ Twine.refreshImmediately = function() {
   }
 };
 
+Twine.register = function(name, constructor) {
```

Comment:
Can you use a different name? `constructor` isn't a reserved word but it's probably better to stay away from it anyways.